oppia/oppia-android
# Fix #387, #1771: Show concept cards

Fixes #387.
Fixes #1771.

Add support for showing concept cards via custom tags at certain points in a lesson. This introduces a new mechanism for handling custom tags. It's been integrated for both concept cards & our list tags; other tags aren't yet integrated (specifically image which is specially handled by Android's HTML parser, anyway). We will hopefully be able to leverage this to simplify #1328, as well.

This is a continuation of #422 and #731 that's been brought up-to-date with ``develop``.

This doesn't add support for concept cards in questions. #1771 is tracking that.

**Testing instructions for explorations:**
- Go to Meaning of Equal parts Exploration
- State 1-> Option D
- State 2-> 3/2 answer
- You will notice concept-card link there.

**Testing instructions for questions:**
- Go to Fractions topic
- Practice tab
- Select Mixed Numbers subtopic
- Start session
- Once you see the question "Amir says the shaded black part of the picture below represents the fraction 1/3" and select the fourth option: "No. Amir has mixed up the numerator and denominator...."
- The concept card link should show up here
